core/include/utils/math_util.h

```diff
@@ -15,4 +16,38 @@ double clamp(double value, double low, double high);
 * 
 * returns the sign +/-1 of x. 0 if x is 0
 **/
-double sign(double x);
\ No newline at end of file
+double sign(double x);
+
+
+/*
+Calculates the variance of  a set of numbers (needed for linear regression)
+https://en.wikipedia.org/wiki/Variance
+@param values   the values for which the covariance is taken
```

Comment:
Comments refer to variance and covariance being taken, which is returned?

Reply:
Oops, that's just a typo it should be variance. 
```suggestion
@param values   the values for which the variance is taken
```